core/src/main/java/psiprobe/tools/ReflectiveAccessor.java

```diff
@@ -54,7 +54,11 @@ public Object get(Object obj, Field field) {
       if (fieldAccessor != null) {
         return get.invoke(fieldAccessor, new Object[] {obj});
       }
-    } catch (Exception ex) {
+    } catch (IllegalArgumentException e) {
```

Comment:
Could also use multi-catch here. We'd have to update the root pom.xml though. Maybe save this for another PR.